### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by anayden**
* Delete unused files
* Separate trainer and torchserve files into their own dirs
* Create base image for both trainer and torchserve to speed up builds

### CODE REVIEW

File: Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/torchserve/Dockerfile [hunk 1]
```diff
@@ -9,25 +9,20 @@ ENV MLFLOW_TRACKING_URI=http://35.185.111.8:5000
...
-COPY requirements.txt /tmp/
 RUN mkdir -p ${MLFLOW_HOME}/scripts && \
     mkdir -p ${MODEL_STORE} && \
     echo "${GCP_CREDS_JSON_BASE64_ENV}"| base64 --decode > "${GOOGLE_APPLICATION_CREDENTIALS}" && \
-    mkdir -p /usr/share/man/man1/ && \
-    apt-get update && apt-get install --no-install-recommends -y git default-jdk wget && \
-    apt-get clean -y -qq && apt-get autoremove -y -qq && \
-    rm -rf /var/lib/apt/lists/* && \
     git clone https://github.com/pytorch/serve.git && \
     cd serve && python ./ts_scripts/install_dependencies.py && \
```
Comment (savagej) [inline Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/torchserve/Dockerfile]: This `install_dependencies.py` script actually installs the correct version of pytorch for your system, e.g. if you have CUDA or even *gasp* windows!??  

I think it makes sense in the base image, and works best run before the `pip install requirements.txt`

Comment (anayden) [reply Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/torchserve/Dockerfile]: So you're suggesting we put TorchServe in both Trainer and Serve images?

Comment (savagej) [reply Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/torchserve/Dockerfile]: Yep, well at least clone the repo and install the dependencies of torchserve.
Probably just makes sense to do the whole lot then I suppose?

Comment (savagej) [reply Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/torchserve/Dockerfile]: Sorry, ignore the whole lot part

Comment (anayden) [reply Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/torchserve/Dockerfile]: Ok, I'll rebuild the base image and update the PR

Comment (anayden) [reply Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/torchserve/Dockerfile]: @savagej done

**Review by savagej — APPROVED** [review]
I went into the branch and looked at the directory 
(╯✧ ∇ ✧)╯
It's beautiful

**Pull request merged**